examples/stable_diffusion/README.md

```diff
@@ -187,3 +187,35 @@ Inference will loop until the generated image. The result will be saved as `resu
 Run `python stable_diffusion.py --help` for additional options. A few particularly relevant ones:
 - `--image_path <str>`: the input image path for image to image inference.
 - `--img_to_img_example`: image to image example. The default input image is `assets/dog.png`, the default prompt is `amazing watercolor painting`.
+
+## Stable Diffusion Quantization encoded in QDQ format
+
+### Generate data for static quantization
+
+To get better result, we need to generate real data from original model instead of using random data for static quantization.
+
+First generate onnx unoptimized model:
+
+`python stable_diffusion.py --model_id stabilityai/sd-turbo --provider qdq --optimize --only_conversion`
```

Comment:
Generating `generate onnx unoptimized model` with `--optimize` option is confusing. Any reason we only need exported model without optimizing?

Reply:
I need this to generate data for static quantization. See the modified OnnxStableDiffusionPipelineWithSave